stdlib/number.gr

```diff
@@ -0,0 +1,143 @@
+import WasmI32 from "runtime/unsafe/wasmi32"
+import WasmI64 from "runtime/unsafe/wasmi64"
+import WasmF64 from "runtime/unsafe/wasmf64"
+import {
+  coerceFloat64,
+  reducedInteger,
+  isFloat
+} from "runtime/numbers"
+import { newFloat64, newInt64 } from "runtime/dataStructures"
+
+primitive (!) : Bool -> Bool = "@not"
+primitive (&&) : (Bool, Bool) -> Bool = "@and"
```

Comment:
Do we need these as primitive definitions here since this file isn't being compiled in a special mode? cc @ospencer

Reply:
Great catch! Should just be able to delete these lines:
```suggestion
```